test/larrays.jl

```diff
@@ -27,10 +27,12 @@ g(x) = x.a
 @inferred g(x)
 
 x = @LArray [1,2,3] (:a,:b,:c)
-x .* x isa LArray
-x .+ 1 isa LArray
-x .+ 1. isa LArray
-eltype(x .+ 1.) === Float64
+@test x .* x isa LArray
+@test x .+ 1 isa LArray
+@test x .+ 1. isa LArray
+z = x .+ ones(Int, 3)
+@test z isa LArray && eltype(z) === Int
```

Comment:
For the purpose of regression test, I think it's better to use mixed precision here
```julia
z = x .+ ones(Float64, 2)
@test z isa LArray && eltype(z) === Float64
```